services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/DynamoDbAsyncTable.java

```diff
@@ -345,56 +345,61 @@
+     *                                                                              .build());
+     * publisher.subscribe(page -> page.items().forEach(item -> System.out.println(item)));
+     * }
+     * <p>
+     * 2) Subscribing to items across all pages
      * <pre>
      * {@code
      *
      * QueryConditional queryConditional = QueryConditional.keyEqualTo(Key.builder().partitionValue("id-value").build());
-     * SdkPublisher<Page<MyItem>> publisher = mappedTable.query(QueryEnhancedRequest.builder()
+     * PagePublisher<MyItem> publisher = mappedTable.query(QueryEnhancedRequest.builder()
      *                                                                              .queryConditional(queryConditional)
      *                                                                              .build());
+     * publisher.subscribe(item -> System.out.println(item));
      * }
      * </pre>
      *
+     * @see DynamoDbAsyncClient#query
      * @param request A {@link QueryEnhancedRequest} defining the query conditions and how
      * to handle the results.
-     * @return a publisher {@link SdkPublisher} with paginated results (see {@link Page}).
+     * @return a publisher {@link PagePublisher} with paginated results (see {@link Page}).
      */
-    default SdkPublisher<Page<T>> query(QueryEnhancedRequest request) {
+    default PagePublisher<T> query(QueryEnhancedRequest request) {
         throw new UnsupportedOperationException();
     }
 
     /**
      * Executes a query against the primary index of the table using a {@link QueryConditional} expression to retrieve a list of
      * items matching the given conditions.
      * <p>
-     * The result is accessed through iterable pages (see {@link Page}) in an interactive way; each time a
-     * result page is retrieved, a query call is made to DynamoDb to get those entries. If no matches are found,
-     * the resulting iterator will contain an empty page. Results are sorted by sort key value in
-     * ascending order by default; this behavior can be overridden in the {@link QueryEnhancedRequest}.
-     * <p>
-     * The additional configuration parameters that the enhanced client supports are defined
-     * in the {@link QueryEnhancedRequest}.
-     * <p>
-     * This operation calls the low-level DynamoDB API Query operation. Consult the Query documentation for
-     * further details and constraints.
-     * <p>
      * <b>Note:</b> This is a convenience method that creates an instance of the request builder avoiding the need to create one
      * manually via {@link QueryEnhancedRequest#builder()}.
      * <p>
      * Example:
      * <pre>
      * {@code
      *
-     * SdkPublisher<Page<MyItem>> publisher =
+     * PagePublisher<MyItem> publisher =
      *     mappedTable.query(r -> r.queryConditional(QueryConditional.keyEqualTo(k -> k.partitionValue("id-value"))));
      * }
      * </pre>
      *
+     * @see #query(QueryEnhancedRequest)
```

Comment:
Should we hint that there are more examples or ways to use this API documented here? This comment applies to all other instances where you've done the same.

Reply:
`@see` tag doesn't really render well with additional info - everything after it will show as a reference link. 

I can cross link all overloaded methods